tests/framework/microvm_helpers.py

```diff
@@ -222,3 +222,26 @@ def run_in_netns(cmd):
 
         # add a route on the host for the clone address
         run(f"ip route add {ingress_ipv4} via {veth_guest_ip}")
+
+    def trace_cmd_guest(self, fns, cmd, port=4321):
+        """Run trace-cmd on the guest, but transfer the data directly to the host."""
+        docker_apt_install("trace-cmd")
```

Comment:
it was added in the previous commit in the image, was it not?

Reply:
Yeah but the idea is we don't regenerate devctr for just this change. So next time we regenerate devctr we can remove this install.

Reply:
that was my guess :)